src/var/isWindow.js

```diff
@@ -0,0 +1,8 @@
+define( function() {
+	"use strict";
+
+	return function isWindow( obj ) {
+		return obj !== null && typeof obj !== "undefined" && obj === obj.window;
```

Comment:
I just removed `obj!=="undefined` and [this test breaks](https://github.com/jquery/jquery/compare/master...ShashankaNataraj:deprecate-is-window#diff-b77bfd25a4e9630e66a502c213900716R173) because 

<img width="522" alt="screen shot 2017-06-24 at 8 27 26 pm" src="https://user-images.githubusercontent.com/650317/27509487-9e76e304-591b-11e7-9e20-3e87611f6be9.png">